packs/equipment-effects/effect-storm-chair.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "uC6KjfiWrTBXYtP8",
-    "img": "systems/pf2e/icons/spells/chain-lightning.webp",
+    "img": "icons/magic/lightning/bolt-strike-blue.webp",
```

Comment:
This icon is different. Not saying it's an issue - just noting it.

Reply:
I made it match with the Storm Chair item (also included with the PR).